models/balancer/balancer_pools_metrics_daily.sql

```diff
@@ -0,0 +1,82 @@
+{{ config(
+    schema = 'balancer',
+    alias = 'pools_metrics_daily',
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'project_contract_address'],
+    incremental_predicates = ['DBT_INTERNAL_DEST.block_date >= date_trunc(\'day\', now() - interval \'7\' day)'],
+    post_hook='{{ expose_spells(\'["arbitrum", "avalanche_c", "base", "ethereum", "gnosis", "optimism", "polygon", "zkevm"]\',
+                            "project",
+                            "balancer",
+                            \'["viniabussafi", "metacrypto"]\') }}'
+    )
+}}
+
+
+WITH 
+trades AS(
+    SELECT 
+        block_date,
+        version,
+        blockchain,
+        project_contract_address,
+        sum(amount_usd) AS swap_amount_usd
+    FROM {{ ref('balancer_trades') }}
+    {% if is_incremental() %}
+    WHERE block_date >= date_trunc('day', now() - interval '7' day)
+    {% endif %}
+    GROUP BY 1, 2, 3, 4
+),
+
+liquidity AS(
+    SELECT
+        day AS block_date,
+        blockchain,
+        version,
+        pool_address AS project_contract_address,
+        pool_type,
+        pool_symbol,
+        sum(pool_liquidity_usd) AS tvl_usd,
+        sum(pool_liquidity_eth) AS tvl_eth
+    FROM {{ ref('balancer_liquidity') }}
+    {% if is_incremental() %}
+    WHERE day >= date_trunc('day', now() - interval '7' day)
```

Comment:
We may use `incremental_predicate` for incremental filters
```suggestion
    WHERE {{incremental_predicate('day')}}
```